### DISCUSSION TITLE
new feature

### DISCUSSION THREAD

**Discussion by augustoaraujoo** [community/community#28666] [Discussions] [answered]
I really wish github had an update to implement chats to make chatting with friends easier :)

**Comment by radar-blip** [comment]
This has been brought up several times before, I'm sure they are considering it. 👍🏼

**Reply by augustoaraujoo** [reply]
a ok thanks :)

**Comment by melvincwng** [comment, marked as answer]
@augustoaraujoo , Great suggestion! Let's hope this gets implemented by the GitHub Team in the future.

**Reply by augustoaraujoo** [reply]
yes !!

**Reply by augustoaraujoo** [reply]
imagine having a new update to add comments on friends projects

**Reply by sumisk91946-art** [reply]
2022/23/08/._forid_sumiii_.Instagram aidi  asamas chat

**Comment by Djibouti972** [comment]
Subject: Feature Suggestion – AI-Powered Natural Language Repo Creation

Hello GitHub Team,

I hope you're doing well!

I have an idea that I believe could be an amazing addition to GitHub's AI tools, especially considering the growing power of GitHub Copilot.

Imagine if users could create repositories simply by describing them in natural language.

For example:
"Create a public repository named `hahaha`, add a `package.json` file, and a `server.js` file with a basic Express server."

GitHub's AI would then automatically:
- Create the repository
- Generate the requested files with the appropriate boilerplate code
- Initialize the project
- Optionally commit the files

This feature could:
- Boost productivity
- Help beginners who struggle with setup
- Save time in prototyping or starting projects

I hope you consider this for future updates. I’d love to see GitHub take repo creation to the next level with AI.

Thank you for your time and amazing work.

Best regards,  
[Mustafa ]

**Comment by Mosugar** [comment]
bsolutely agree!
Real-time or even threaded messaging on GitHub would be a huge help, especially for open source collaborators and team discussions.
For now, I’ve seen some devs integrate tools like Slack, Discord, or even Gitter with their repos — but a native GitHub chat would be amazing 
Hope the GitHub team picks this idea up soon!
